src/gui/SearchWidget.cpp

```diff
@@ -76,14 +75,9 @@ bool SearchWidget::eventFilter(QObject *obj, QEvent *event)
             }
         }
         else if (keyEvent->matches(QKeySequence::MoveToNextLine)) {
-            // If Down is pressed at EOL in the search edit, move
-            // the focus to the entry view.
-            QLineEdit* searchEdit = m_ui->searchEdit;
-            if (!searchEdit->hasSelectedText() &&
-                searchEdit->cursorPosition() == searchEdit->text().length()) {
-                emit downPressed();
-                return true;
-            }
+            // If Down is pressed move the focus to the entry view.
```

Comment:
On most of applications on GNU/Linux the Up/Down is used to going at EOL. On QLineEdit this behaviour isn't followed by default so on GNU/Linux you have to use Ctrl+Right.

I think pressing Down in the middle of the text is not meant to be used as "set focus on entryView" but instead for placing pointer to EOL. 
When pointer is at EOL we can change the focus to entryView